my_project/template_graphs.py

```diff
@@ -149,16 +149,21 @@ def yearly_profile(df, var, global_local, si_ip):
 
     if var == "DBT":
         # plot ashrae adaptive comfort limits (80%)
+        map80= dbt_day["mean"]
         lo80 = df.groupby("DOY")["adaptive_cmf_80_low"].mean().values
         hi80 = df.groupby("DOY")["adaptive_cmf_80_up"].mean().values
+        # set color https://github.com/CenterForTheBuiltEnvironment/clima/issues/113 implemention 
+        var_barcolors =np.where((map80 > 40) | (map80 < 10), 'lightgray', 'darkgray')
```

Comment:
I think this may be incorrect, you should filter on the running mean temperature and not based on the dry-bulb temperature.

Reply:
Yes, I was referring to the `running_mean_outdoor_temperature`. The right way of doing it is to add the variable `rmt` to a new column in the data frame and then use this variable when filtering the data in the plot. We can discuss this today.

Reply:
As we discuss today, the issues should be all set. I delete the line for the range of x and the range of rmt. Then add the variable rmt to a new column in the data frame to filter the data.